src/libostree/ostree-deployment.c

```diff
@@ -27,12 +27,24 @@ typedef GObjectClass OstreeDeploymentClass;
 
 G_DEFINE_TYPE (OstreeDeployment, ostree_deployment, G_TYPE_OBJECT)
 
+/*
+ * ostree_deployment_get_csum:
+ * @self: Deployemnt
+ *
+ * Returns: (not nullable): The OSTree commit used for this deployment.
+ */
 const char *
 ostree_deployment_get_csum (OstreeDeployment *self)
 {
   return self->csum;
 }
 
+/*
+ * ostree_deployment_get_bootcsum:
+ * @self: Deployemnt
```

Comment:
Here and multiple times below:

```suggestion
 * @self: Deployment
```